### PR TITLE
Hydra url as /ws (proxied in nginx)

### PR DESCRIPTION
**Pull request by algreasley**
See corresponding deployments PR: https://github.com/AdaptiveConsulting/reactive-deployments/pull/3/files

### CODE REVIEW

File: src/new-client/src/Finsemble/index.tsx [hunk 1]
```diff
@@ -10,7 +10,7 @@ import { noop } from "rxjs"
 
 if (!import.meta.env.VITE_MOCKS) {
   connectToGateway({
-    url: import.meta.env.VITE_HYDRA_URL as string,
+    url: "/ws",
```
Comment (algreasley) [inline src/new-client/src/Finsemble/index.tsx:13]: _not complete_

File: src/new-client/src/OpenFin/index.tsx [hunk 2]
```diff
@@ -10,7 +10,7 @@ import { noop } from "rxjs"
 
 if (!import.meta.env.VITE_MOCKS) {
   connectToGateway({
-    url: import.meta.env.VITE_HYDRA_URL as string,
+    url: "/ws",
```
Comment (algreasley) [inline src/new-client/src/OpenFin/index.tsx:13]: _not complete_

File: src/new-client/src/Web/index.tsx [hunk 3]
```diff
@@ -15,7 +15,7 @@ import { noop } from "rxjs"
 
 if (!import.meta.env.VITE_MOCKS) {
   connectToGateway({
-    url: import.meta.env.VITE_HYDRA_URL as string,
+    url: `${window.location.href}ws`,
```
Comment (algreasley) [inline src/new-client/src/Web/index.tsx:18]: _not complete_  just added to test quickly

File: src/new-client/vite.config.ts [hunk 4]
```diff
@@ -124,6 +124,12 @@ const setConfig = ({ mode }) => {
     },
     server: {
       port: 1917,
+      proxy: !process.env.VITE_MOCKS && {
+        "/ws": {
+          target: "http://localhost:55000",
```
Comment (algreasley) [inline src/new-client/vite.config.ts:129]: we need this locally, as we have to run through the nginx running in Docker with its associated config - going direct to wss:<demo WS url> does not work as I think we need those special proxy args - we may be able to apply them to the local vite (http-proxy) config - any thoughts, @garyadaptive ?

Comment (ivanfalanga) [inline src/new-client/vite.config.ts:129]: Where does this url come from exactly? port 55000 sounds oddly specific, was this something you mapped to locally to connect to the locally running container of the gateway?

Comment (algreasley) [reply src/new-client/vite.config.ts:129]: Docker - just updating this now so the default (if no VITE_MOCKS) will route through to our one and only Hydra endpoint (demo)

**Review by ivanfalanga — APPROVED** [review]

**Pull request merged**